addons/website/views/snippets/snippets.xml

```diff
@@ -550,10 +550,10 @@
 
     <div data-js="CoverProperties" data-selector=".o_record_cover_container" data-no-check="true">
         <we-imagepicker string="Image" data-background="" data-name="cover_background_opt"/>
-        <we-select string="Size" data-cover-opt-name="size">
-            <we-button data-select-class="o_record_has_cover o_full_screen_height">Full Screen</we-button>
-            <we-button class="o_record_cover_opt_size_default" data-select-class="o_record_has_cover o_half_screen_height">Half Screen</we-button>
-            <we-button data-select-class="o_record_has_cover cover_auto">Fit text</we-button>
+        <we-select string="Size" data-name="cover_background_size">
+            <we-button data-select-class="o_full_screen_height">Full Screen</we-button>
+            <we-button class="o_record_cover_opt_size_default" data-select-class="o_half_screen_height">Half Screen</we-button>
+            <we-button data-select-class="cover_auto">Fit text</we-button>
```

Comment:
Can you explain those change to me? I feel it is breaking everything but there should be a reason I don't know?